### DISCUSSION TITLE
Welcome to mialsuperresolutiontoolkit Discussions!

### DISCUSSION THREAD

**Discussion by sebastientourbier** [Medical-Image-Analysis-Laboratory/mialsuperresolutiontoolkit#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrzaib** [comment]
> ##  Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together muscle.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

my issue resolved..?

**Reply by mrzaib** [reply]
well my name is zaib amd i dont belong to family who can afford my mobile for me so i got issue and i was searcjing for any soluotion so i came here  ... kindly tell me if my problem is resolved so should i do up next
